src/App.vue

```diff
@@ -228,12 +230,31 @@ watch(
   }
 )
 
+
+// === Score Chords functions === 
+function pushSymbolToChord(index: number){
+  state.scoreChord.push(scale.symbols[index])
+}
+
+function pullSymbolfromChord(index: number){
+  const item = state.scoreChord.indexOf(scale.symbols[index])
+  state.scoreChord.splice(item, 1)
```

Comment:
Does `scoreChord` really need to be an `Array`? I don't think the order matters in a vertically stacked chord so a `Set` makes more sense semantically.